package.json

```diff
@@ -26,6 +33,9 @@
     "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000"
   },
   "dependencies": {
-    "hapi": "^16.6.0"
+    "bookshelf": "^0.10.4",
+    "hapi": "^16.6.0",
+    "knex": "^0.13.0",
+    "pg": "^7.3.0"
```

Comment:
Nit: alphabetize the `package.json`, so move this up above `devDependencies`, etc.